src/v2/view-builder/components/Link.js

```diff
@@ -21,11 +21,14 @@ export default View.extend({
 
   attributes() {
     let href = this.options.href || '#';
+    const rel = (this.options.target && !this.options.rel)
+      ? 'noopener noreferrer'
+      : undefined;
     return {
       'data-se': this.options.name,
       href: href,
       target: this.options.target,
-      rel: this.options.rel
+      rel: rel
```

Comment:
@glenfannin-okta in response to your comment/concern on https://github.com/okta/okta-signin-widget/pull/3721